server/pairing/raw_message_collector.go

```diff
@@ -21,13 +21,13 @@ func (r *RawMessageCollector) getRawMessages() []*common.RawMessage {
 	return r.rawMessages
 }
 
-func (r *RawMessageCollector) convertToSyncRawMessage() *protobuf.SyncRawMessage {
+func (r *RawMessageCollector) convertToSyncRawMessage() protobuf.SyncRawMessage {
```

Comment:
can you explain why not use pointer , thank you 🙂

Reply:
Ah, actually this doesn't need to be a value now. I previously during this refactor had code that passed out a value and assigned to a pointer's value. But during this refactor, I realised that `protobuf.SyncRawMessage` is the transport wrapper and not the actual payload, so I changed the refactor to use a `RawMessagePayload` as the payload for loading, storing and marshalling.

So I will change this back to a pointer, because it is completely fine for it to be a pointer.